requirements.txt

```diff
@@ -2,7 +2,7 @@ dynaconf==3.1.12
 fastapi==0.99.0
 PyGithub==1.59.*
 retry==0.9.2
-openai==0.27.8
+openai==1.3.5
```

Comment:
**Suggestion:** It's a good practice to specify the minimum required version for each dependency in your `requirements.txt` file. This ensures that your project will not break due to updates in your dependencies. For example, instead of `openai==1.3.5`, you could use `openai>=1.3.5`. [best practice]
```suggestion
openai>=1.3.5
```